components/style/properties/gecko.mako.rs

```diff
@@ -1921,6 +1919,18 @@ clip-path
     }
 
     ${impl_coord_copy('column_width', 'mColumnWidth')}
+
+    pub fn set_column_count(&mut self, v: longhands::column_count::computed_value::T) {
+        use std::cmp::min;
```

Comment:
we already import `std::cmp` at the top of the file. I think it looks cleaner just using `cmp::min` and dropping this import.

Also, a match might be a better bet for this instead of `if let`, but that's arguable:

```rust
self.gecko.mColumnCount = match v.0 {
    Some(number) => cmp::min(number, nsStyleColumn_kMaxColumnCount),
    None => NS_STYLE_COLUMN_COUNT_AUTO,
};
```

r=me with the first, and optionally with the second suggestion.